src/Sentry/Sentry.csproj

```diff
@@ -8,7 +8,7 @@
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(SolutionName)' != 'Sentry.Unity'">
-    <TargetFrameworks>net8.0;net6.0;netstandard2.1;netstandard2.0;net462</TargetFrameworks>
+    <TargetFrameworks>net8.0;net6.0;netstandard2.1;netstandard2.0;net462;net48</TargetFrameworks>
```

Comment:
Is this supposed to be a new target or to replace the net462 target?

Reply:
No, we need to keep supporting the older one, but we can at least improve for apps running on the latest version